docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
-```
+2. Find the latest version, and download the file ending in `-windows-amd64.zip`.
 
-You can use the Offer, Publisher and Sku to pick a specific version by adding `windowsOffer`, `windowsPublisher`, `windowsSku` and (optionally) `widndowsVersion` to the `windowsProfile` section. In this example, the latest Windows Server version 1803 image would be deployed.
+3. Extract the `acs-engine...-windows-amd64.zip` file to a working folder such as `c:\tools`
 
-```
-"windowsProfile": {
-            "adminUsername": "azureuser",
-            "adminPassword": "...",
-            "windowsPublisher": "MicrosoftWindowsServer",
-            "windowsOffer": "WindowsServerSemiAnnual",
-            "windowsSku": "Datacenter-Core-1803-with-Containers-smalldisk"
-     },
+4. Check that it runs with `.\acs-engine.exe version`
+
+```none
+PS C:\Users\patrick\acs-engine> .\acs-engine.exe version
+Version: v0.20.6
+GitCommit: 293adfda
+GitTreeState: clean
 ```
 
-## Walkthrough
+5. [optional] Add the folder you created in step 3 to your path.
 
-Once your Kubernetes cluster has been created you will have a resource group containing:
+```powershell
+$ENV:Path += ';c:\tools'
+# If you want to save the setting permanently, then run
+$oldPath = [Environment]::GetEnvironmentVariable('Path', [EnvironmentVariableTarget]::User)
+[Environment]::SetEnvironmentVariable('Path', $oldPath + ';c:\tools', [EnvironmentVariableTarget]::User)
+```
 
-1. 1 master accessible by SSH on port 22 or kubectl on port 443
+#### Kubectl for Windows
 
-2. a set of windows and linux nodes.  The windows nodes can be accessed through an RDP SSH tunnel via the master node.  To do this, follow these [instructions](../ssh.md#ssh-to-the-machine), replacing port 80 with 3389.  Since your windows machine is already using port 3389, it is recommended to use 3390 to Windows Node 0, 10.240.0.4, 3391 to Windows Node 1, 10.240.0.5, and so on as shown in the following image:
+The latest release of Kubernetes Control (kubectl) is available on the [Kubernetes release page](https://kubernetes.io/docs/imported/release/notes/). Look for `kubernetes-client-windows-amd64.tar.gz` and download it.
 
-![Image of Windows RDP tunnels](../images/rdptunnels.png)
+Windows 10 version 1803 already includes `tar`, so extract the archive and move `kubectl.exe` to the same folder (such as `c:\tools`) that you put `acs-engine.exe`. If you don't already have `tar`, then [busybox-w32](https://frippery.org/busybox/) is a good alternative. Download [busybox.exe](https://frippery.org/files/busybox/busybox.exe), then copy it to `c:\tools\tar.exe`. It must be named to `tar.exe` for the next step to work.
 
-The following image shows the architecture of a container service cluster with 1 master, and 2 agents:
+```powershell
+tar xvzf C:\Users\patrick\Downloads\kubernetes-client-windows-amd64.tar.gz
+Move-Item .\kubernetes\client\bin\kubectl.exe c:\tools
+```

```

Comment:
Same as previous comment on line 76, mention alias `mv` for line 80 as an option for *nix